Makefile

```diff
@@ -1,30 +1,9 @@
-COQWEB=~/COQ/www/
-SUITES_COQPKGIDX = released
-SUITES= $(SUITES_COQPKGIDX) core-dev extra-dev
+SUITES= released core-dev extra-dev
 H=@
-COQV=8.8.2
-OCAMLV=4.02.3
-
-pp = (cd $(COQWEB); yamlpp-0.3/yamlpp -l en $(abspath $(1)) -o $(abspath $(2)))
-
-ifeq "$(shell test ! -z '$(COQWEB)' -a -d $(COQWEB) || echo false)" "false"
-$(error "Please use 'make COQWEB=path/to/coq/www'")
-endif
 
 # refresh opam indexes + generate website
```

Comment:
This comment is also a bit outdated

Reply:
Good point. I will fix it and then commit, since you already approved.